### ISSUE TITLE
Planet Cli2 SDK-Data API: Search command is giving error as invalid item type for a valid one

### ISSUE THREAD

**Issue by rajashekarmamidala** [planetlabs/planet-client-python#682]
**Environment**: planet, version 2.0a2

**Test steps**:
1. Do a search with data api
`planet data search --limit 2 item_types PSScene`

**Expected**: It should display the data for PSScene 

**Actual**: It is displaying error as below

```
Usage: planet data search [OPTIONS] ITEM_TYPES [FILTER]
Try 'planet data search --help' for help.

Error: Invalid value for 'ITEM_TYPES': ['item_types'] should be one of {'REOrthoTile', 'SkySatCollect', 'PSScene4Band', 'PSOrthoTile', 'Sentinel1', 'SkySatScene', 'Sentinel2L1C', 'Landsat8L1G', 'PSScene3Band', 'MYD09GQ', 'MOD09GA', 'PSScene', 'REScene', 'MOD09GQ', 'MYD09GA'}
```

**Comment by cholmes** [comment]
Thanks for the report @rajashekarmamidala! 

So for this one the proper is `planet data search --limit 2 PSScene` - you don't include the `item_types` word in the request.

You still need to include a filter though - if you just enter the above it'll hang, waiting for a filter.

**Comment by jreiberkyle** [comment]
fixed with #840
